docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
+[main] c.r.c.host.web.EnclaveWebHost$Companion  : Started EnclaveWebHost.Companion in <SECONDS> seconds 
 ```
 
-!!! warning
-    The sample uses Conclave mail which handles encryption and authentication, but the web host protocol makes use of
-    request headers which are not encrypted. Because of this, it is still important to use HTTPS for anything other than
-    internal development. Setting up an HTTPS connection is beyond the scope of this tutorial, but you may wish to look
-    at configuring Spring Boot or setting up a reverse proxy such as Nginx or Apache. See
-    [Conclave web host](conclave-web-host.md) for more information.
+!!!Warning
+
+    The sample uses Conclave Mail which handles encryption and authentication. But the web host protocol uses
+    unencrypted request headers. So, it is important to use HTTPS for anything other than internal development.
+    Setting up an HTTPS connection is beyond the scope of this tutorial. You can try configuring Spring Boot or 
+    setting up a reverse proxy such as Nginx or Apache. See [Conclave web host](conclave-web-host.md) for more 
+    information.
 
 ## Run the client
 
-Run the client with
+After running the host:
+
+1. Open another command line interface.
+2. Navigate to the hello-world sample.
 ```bash
-java -jar client/build/libs/client.jar \
-  # A constraint which defines which enclaves the client will accept.
-  "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" \
-  # The string to reverse
-  reverse-me
+cd conclave-tutorials/hello-world
 ```
+3. Run the client with the command:
+```bash
+java -jar client/build/libs/client.jar "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1
+SEC:INSECURE" reverse-me
+```
+In the above command, the parameters in quotes are the constraints, and 'reverse-me' is the string to be reversed.
 
-You should see the output
+You will see the output:
 ```bash
 Reversing `reverse-me` gives `em-esrever`
 ```
-which has been produced by the following sequence of events:
-
-1. The host loaded the enclave, which generated a remote attestation report.
-2. The client connected to the host and downloaded the attestation report.
-3. The client used the attestation report to verify that it was connected to the enclave it expected, and that the
-   enclave was running in the expected mode.
-4. The client encrypted the "reverse-me" string using the public key provided in the attestation.
-5. As part of the message to the enclave, the client also included a public key for the enclave to use when responding.
-6. The client sent the encrypted message to the host's REST API.
-7. The host passed the encrypted message to the enclave.
-8. The enclave decrypted the client's message, reversed the string, and encrypted the response using the client's
+
+You have run your first enclave successfully.
+
+The following sequence of events produces the above output:
```

Comment:
This is very important explanation, I would highlight the 'title' somehow.

Reply:
I agree with @anien , this deserves a title.
I would also personally make emphasis on the "entities" (host/client/enclave). Not sure if it's a good idea to have them upper cases or make them bold.
As this is a "software use case" one convention is to have entities underlined and the action in bold.
I.e. something like

The <ins>host</ins> **loads** the <ins>enclave</ins>, which **generates** a remote attestation report.

Not sure if this is nice to see though, you're the expert :-)

Reply:
I have added a new subheading. 

I wouldn't add underlines or bold here as it will go against our style guide. This is an ideal piece of content to make an architectural diagram, or even an explainer video. Will mark for such an upgrade later.